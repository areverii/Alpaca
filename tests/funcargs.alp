function void add_and_print(int x, int y):
    # We don't really need an extra variable here but we want to test the vars
    int z
    z = x + y
    printi(z)
end

function void print_string(string s):
    prints("In print_string:")
    prints(s)
end

function void main(<string> args):
    add_and_print(3, -4)
    print_string("test_string")
end
