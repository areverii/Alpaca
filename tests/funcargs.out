-1
In print_string:
test_string
